shared/common-adapters/button.tsx

```diff
@@ -78,13 +78,13 @@ class Button extends React.Component<Props> {
 
     containerStyle = Styles.collapseStyles([containerStyle, this.props.style])
 
-    const onClick = (!unclickable && this.props.onClick) || null
+    const onClick = (!unclickable && this.props.onClick) || undefined
```

Comment:
my goal for these prs was (when possible) to try and change it as little as possible to get the typing to be correct and avoid little bugs as i was moving stuff around. my hope is the ugliness thats left around like this will be annoying and be fixed as we revisit these files